### PR TITLE
[zh-cn]: Update translation of Function.call()

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

Same as the title.

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#15403

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/function/call/index.md [hunk 1]
```diff
@@ -5,127 +5,109 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/call
...
 ```
 
 ### 参数
 
 - `thisArg`
-  - : 可选的。在 _`function`_ 函数运行时使用的 `this` 值。请注意，`this`可能不是该方法看到的实际值：如果这个函数处于[非严格模式](/zh-CN/docs/Web/JavaScript/Reference/Strict_mode)下，则指定为 `null` 或 `undefined` 时会自动替换为指向全局对象，原始值会被包装。
-- `arg1, arg2, ...`
-  - : 指定的参数列表。
+  - : 在调用 `func` 时要使用的 `this` 值。如果函数不在[严格模式](/zh-CN/docs/Web/JavaScript/Reference/Strict_mode)下，[`null`](/zh-CN/docs/Web/JavaScript/Reference/Operators/null) 和 [`undefined`](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/undefined) 将被替换为全局对象，并且原始值将被转换为对象。
+- `arg1`, …, `argN` {{optional_inline}}
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/function/call/index.md:25]: align with [bind](https://github.com/mdn/translated-content/blob/41b3960fa9eeef4ef8f0fec0420ad7a6dbd2cc7e/files/zh-cn/web/javascript/reference/global_objects/function/bind/index.md?plain=1#L25)

```suggestion
- `arg1, …, argN` {{optional_inline}}
```

File: files/zh-cn/web/javascript/reference/global_objects/function/call/index.md [hunk 2]
```diff
@@ -5,127 +5,109 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/call
...
 
 ## 示例
 
-### 使用 `call` 方法调用父构造函数
+### 使用 call() 调用函数并指定 this 值
 
-在一个子构造函数中，你可以通过调用父构造函数的 `call` 方法来实现继承，类似于 `Java` 中的写法。下例中，使用 `Food` 和 `Toy` 构造函数创建的对象实例都会拥有在 `Product` 构造函数中添加的 `name` 属性和 `price` 属性，但 `category` 属性是在各自的构造函数中定义的。
+在下面的示例中，当我们调用 `greet` 时，`this` 的值将绑定到对象 `obj`，即使 `greet` 不是 `obj` 的方法。
 
 ```js
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/function/call/index.md:46]: The sentence should be:

```
猫的睡眠时间一般在 12 到 16 小时之间
```

File: files/zh-cn/web/javascript/reference/global_objects/function/call/index.md [hunk 3]
```diff
@@ -5,127 +5,109 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/call
...
-  }).call(animals[i], i);
+globalThis.globProp = "Wisen";
+
+function display() {
+  console.log(`globProp 的值是 ${this.globProp}`);
 }
-```
 
-### 使用 `call` 方法调用函数并且指定上下文的 '`this`'
+display.call(); // Logs "globProp 的值是 Wisen"
```
Comment (jasonren0403) [inline files/zh-cn/web/javascript/reference/global_objects/function/call/index.md:70]:
```suggestion
display.call(); // 输出“globProp 的值是 Wisen”
```

**Review by jasonren0403 — COMMENTED** [review]
some suggestions

File: files/zh-cn/web/javascript/reference/global_objects/function/call/index.md [hunk 4]
```diff
@@ -5,127 +5,109 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/call
...
-function Product(name, price) {
-  this.name = name;
-  this.price = price;
-}
-
-function Food(name, price) {
-  Product.call(this, name, price);
-  this.category = "food";
+function greet() {
+  console.log(this.animal, "通常在", this.sleepDuration, "之间睡觉");
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/call/index.md:48]:
```suggestion
  console.log(this.animal, "的睡眠时间一般在", this.sleepDuration, "之间");
```

File: files/zh-cn/web/javascript/reference/global_objects/function/call/index.md [hunk 5]
```diff
@@ -5,127 +5,109 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/call
...
-  this.category = "toy";
-}
+const obj = {
+  animal: "猫",
+  sleepDuration: "12 到 16 点",
+};
 
-var cheese = new Food("feta", 5);
-var fun = new Toy("robot", 40);
+greet.call(obj); // 猫 通常在 12 到 16 点 之间睡觉
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/call/index.md:56]:
```suggestion
greet.call(obj); // 猫 的睡眠时间一般在 12 到 16 小时 之间
```

File: files/zh-cn/web/javascript/reference/global_objects/function/call/index.md [hunk 6]
```diff
@@ -5,127 +5,109 @@ slug: Web/JavaScript/Reference/Global_Objects/Function/call
...
+  console.log(this.animal, "通常在", this.sleepDuration, "之间睡觉");
 }
 
-function Toy(name, price) {
-  Product.call(this, name, price);
-  this.category = "toy";
-}
+const obj = {
+  animal: "猫",
+  sleepDuration: "12 到 16 点",
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/function/call/index.md:53]:
```suggestion
  sleepDuration: "12 到 16 小时",
```

**Review by jasonren0403 — APPROVED** [review]
👍 Thanks @JasonLamv-t

**Pull request merged**